### ISSUE TITLE
Support LngLat object in Marker coordinates

### ISSUE THREAD

**Issue by iBobik** [soal/vue-mapbox#176]
Current coordinates props supports only array, but LngLat works also, but is not allowed by the type check:

[Vue warn]: Invalid prop: type check failed for prop "coordinates". Expected Array, got Object 
found in ---> <MapMarker>

**Comment by richardfeliciano** [comment]
Yep, i`m trying to center markers, and always get this warn, is annoying,

**Comment by danimalweb** [comment]
I also have this issue.
